.github/workflows/release.yml

```diff
@@ -41,6 +41,25 @@ jobs:
           python setup.py sdist bdist_wheel
           twine upload dist/*
 
+  publish-python-sdk-docker:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+    needs:
+      - publish-python-sdk
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build Merlin SDK Docker py${{ matrix.python-version }}
+        working-directory: ./python/sdk
+        run: |
+          IMAGE_TAG="ghcr.io/caraml-dev/merlin-sdk:py${{ matrix.python-version }}-${{ inputs.version }}"
```

Comment:
Shall we define the `CONTAINER_REGISTRY: ${{ env.DOCKER_REGISTRY }}/${{ github.repository }}` variable common to all jobs and create this `IMAGE_TAG` variable based on that? We are already doing this for some jobs like `publish-pyfunc`.